apps/orgs/utils.py

```diff
@@ -57,6 +57,8 @@ def get_org_from_request(request):
 def set_current_org(org):
     if isinstance(org, (str, uuid.UUID)):
         org = Organization.get_instance(org)
+        if not org:
+            return
     setattr(thread_local, 'current_org_id', org.id)
 
 
```

Comment:
Sorry, but it appears you have submitted irrelevant information for this request. Can you please provide the original code and I would be happy to take a look at its current version? Also, which part of the code needs checking for differences?